src/test/javascript/spec/component/plagiarism/plagiarism-inspector.component.spec.ts

```diff
@@ -47,29 +46,34 @@ describe('Plagiarism Inspector Component', () => {
     const activatedRoute = {
         data: of({ exercise: modelingExercise }),
     };
-    const plagiarismResult = {
-        comparisons: [
-            {
-                submissionA: { studentLogin: 'student1A' },
-                submissionB: { studentLogin: 'student1B' },
-                similarity: 0.5,
-                status: PlagiarismStatus.NONE,
-            },
-            {
-                submissionA: { studentLogin: 'student2A' },
-                submissionB: { studentLogin: 'student2B' },
-                similarity: 0.8,
-                status: PlagiarismStatus.NONE,
-            },
-            {
-                submissionA: { studentLogin: 'student3A' },
-                submissionB: { studentLogin: 'student3B' },
-                similarity: 0.7,
-                status: PlagiarismStatus.NONE,
-            },
-        ],
+    const comparisons = [
+        {
+            submissionA: { studentLogin: 'student1A' },
+            submissionB: { studentLogin: 'student1B' },
+            similarity: 0.5,
+            status: PlagiarismStatus.NONE,
+        },
+        {
+            submissionA: { studentLogin: 'student2A' },
+            submissionB: { studentLogin: 'student2B' },
+            similarity: 0.8,
+            status: PlagiarismStatus.NONE,
+        },
+        {
+            submissionA: { studentLogin: 'student3A' },
+            submissionB: { studentLogin: 'student3B' },
+            similarity: 0.7,
+            status: PlagiarismStatus.NONE,
+        },
+    ];
+    const modelingPlagiarismResult = {
+        comparisons,
     } as ModelingPlagiarismResult;
 
+    const textPlagiarismResult = {
+        comparisons,
```

Comment:
Does it make sense that these are the same objects? Would a deepClone make sense here?